src/container/__tests__/RelayRenderer_render-test.js

```diff
@@ -45,31 +59,31 @@ describe('RelayRenderer.render', () => {
         return this.actual.props.shouldUpdate;
       },
       toBeShallowRenderedChild() {
```

Comment:
If we're going to stick with a change to the deep renderer, let's rename these matchers.